arches/app/media/css/arches.scss

```diff
@@ -12857,8 +12857,10 @@ ul.select2-choices:after {
 }
 
 .map-coordinate-editor-list {
-    padding: 0 5px;
-    margin-top: 70px;
+    padding-top: 70px;
+    padding-left: 5px;
+    height: 250px;
```

Comment:
Could you increase the height to around 350px?

Reply:
Resolved in the latest commmit [1d812e4](https://github.com/archesproject/arches/pull/10080/commits/1d812e43a6668f4f152484f3c7ed874ed638fc30)